src/lib/Microsoft.Health.Fhir.Ingest/FhirHealthCheckStatus.cs

```diff
@@ -0,0 +1,20 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+namespace Microsoft.Health.Fhir.Ingest
```

Comment:
If this is being used to record the outcome the health check it probably makes sense to be in the Data namespace & folder.

Reply:
Makes sense. Moved